apps/web-giddh/src/app/settings/integration/setting.integration.component.html

```diff
@@ -622,10 +622,12 @@ <h3 class="section-head mb-2">Shopify</h3>
                             <!-- New code -->
                             <div>
                                 <div class="form-group update-btn-wrapper">
-                                    <label>&nbsp;</label>
-                                    <br>
-                                    <button type="button" class="btn btn-success mr-r1" [disabled]="!paymentFormUpdate.form.valid" (click)="updateIciciDetails(regAcc, parentIndex)">Update
-                                </button>
+                                    <label *ngIf="isBankUpdateInEdit === parentIndex">&nbsp;</label>
```

Comment:
Consider moving label and br tags to ```ng-container``` as they have same ngIf condition `isBankUpdateInEdit === parentInde`